integrations/docker/images/chip-cirque-device-base/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:focal
+
+ARG OT_BR_POSIX_CHECKOUT=master
```

Comment:
master checkouts mean that every docker build potentially generates a different version.

How about fixing the version to a SHA, so docker builds are reproducible?

Reply:
The SHA is set by build.sh.